.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,48 @@
- **JIRA**: (link to JIRA ticket)
- **Linked PRs**: (links to corresponding PRs, optional)
+<!--- Provide a general summary of your changes in the Title above -->
 
-## How to test-drive this PR
-- (step1)
+# Description
 
-## Changes
-- (change1)
+<!--- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. -->
 
-## Todos
+# Types of changes
 
-### General
-- [ ] Add a high-level description of your changes to relevant CHANGELOG.md files (**not** required for doc updates)
-- [ ] The code changes are covered by test cases
-- [ ] Analytics events instrumented to measure impact of change
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
 
-### Backwards Compatibility
-- [ ] This PR contains no breaking changes or breaking changes have been approved by Product Management/Engineering as necessary.
+- [ ] **Bug fix** (non-breaking change which fixes an issue)
+- [ ] **New feature** (non-breaking change which adds functionality)
+- [ ] **Documentation update**
```

Comment:
Is there still enough documentation in the repo to warrant its own checkbox? The only forms of documentation in the repo I can think of are the three README.md files and comments within the code (including the JSDoc comments).